ManikShrivastav.md

```diff
@@ -0,0 +1,45 @@
+---
+Name: “ManikShrivastav”
+Project: “Gesture Robot”
+---
+
+# Project Name
```

Comment:
Could you replace this title with the name of your project?

Reply:
Done!